dev/io.openliberty.security.common.jwt/src/io/openliberty/security/common/jwt/jws/JwsVerificationKeyHelper.java

```diff
@@ -86,10 +87,10 @@ String getSignatureAlgorithmFromJws(JsonWebStructure jws) throws UnsupportedSign
     }
 
     Key getSharedKey() throws SharedKeyMissingException, UnsupportedEncodingException {
-        if (clientSecret == null || clientSecret.isEmpty()) {
+        if (sharedSecret == null || sharedSecret.isEmpty()) {
             throw new SharedKeyMissingException();
         }
-        String clientSecretProtectedString = new String(clientSecret.getChars());
+        String clientSecretProtectedString = new String(sharedSecret.getChars());
```

Comment:
```suggestion
        String sharedSecretProtectedString = new String(sharedSecret.getChars());
```
I can come back and update this later.